src/store/reducers/userSlice.ts

```diff
@@ -0,0 +1,73 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export class UserState {
+  user?: {
```

Comment:
User don't have any required fields?

Reply:
But the user can be defined with any fields